### DISCUSSION TITLE
Missing top layer for teammates

### DISCUSSION THREAD

**Discussion by MUGEN-Slayer** [ikemen-engine/Ikemen-GO#1299] [Feature Request] [answered]
### Describe the bug

While all lifebar elements received the extra "top" bg, this was missing for the teammates' portraits. While I believe that top as its name implies should always have the highest priority. In this specific case, I think ko.spr/anim should have higher priority, as both are customizable.

### To Reproduce

px.teammate.top.spr =
px.teammate.top.anim = 

### Engine Version (or source code date)

v0.99.0-rc.1

### Operating system

Windows

**Comment by K4thos** [comment, marked as answer]
added by #3992
